doc/userguide/rules/ldap-keywords.rst

```diff
@@ -0,0 +1,44 @@
+LDAP Keywords
+=============
+
+.. role:: example-rule-action
+.. role:: example-rule-header
+.. role:: example-rule-options
+.. role:: example-rule-emphasis
+
+ldap.request.operation
+----------------------
+
+Suricata has a ``ldap.request.operation`` keyword that can be used in signatures to identify
+and filter network packets based on Lightweight Directory Access Protocol request operations.
+
+Syntax::
+
+ ldap.request.operation: operation;
+
+ldap.request.operation uses :ref:`unsigned 8-bit integer <rules-integer-keywords>`.
+
+.. table:: **Operation values for ldap.request.operation keyword**
+
+    ====  ================================================
```

Comment:
When you implement ldap.response.operation, the same table should be used for both